runtime/src/lib.rs

```diff
@@ -245,8 +268,20 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u128 = 500;
-    pub const MaxLocks: u32 = 50;
+	pub const UncleGenerations: u32 = 0;
+}
+
+impl pallet_authorship::Config for Runtime {
+	type FindAuthor = ();
```

Comment:
let's put a TODO in here to make sure we add in FindAccountFromAuthorIndex when aura gets used